lms/djangoapps/learning_success/tasks.py

```diff
@@ -0,0 +1,49 @@
+
+from celery import task
+from celery_utils.logged_task import LoggedTask
+
+from learning_success.export_all_activity_records import export_all_activity_records as perform_activity_export  # noqa
+from learning_success.export_coding_challenge_data import (
+    CodeChallengeExporter)
+
+
+@task(base=LoggedTask)
+def export_all_activity_records(source_platform, pathway, programme_ids):
+    """ POST the collected data to the api endpoint from the settings
+        Arguments:
+            source_platform: Platform import as, i.e. 'juniper' or 'ginkgo'
+            programme_ids: Programme ids of programme to use 'disd'
+
+    The table should have one entry per day per platform and programme
+    per student.
+
+    In order to be able to easily re-run the extract any given day
+    without any further intervention we are deleting any existing
+    entry first.
+
+    With the use of one transaction for deletion and insertion we can
+    make sure that one does not happen without the other as to not
+    lose any information.
+    """
+    perform_activity_export(source_platform, pathway, programme_ids)
```

Comment:
`programme_ids` can be multiple args, will this still work?
We currently run it like so:
`python manage.py lms export_all_activity_records juniper fullstack disd disde diwad`
Whereby `disd disde diwad` are three `programme_ids`